scans/auth/plans_and_scripts/authtesttotp/bbaplus.yaml

```diff
@@ -0,0 +1,78 @@
+env:
+  contexts:
+  - name: Authentication Test
+    urls:
+    - https://authenticationtest.com/
+    includePaths:
+    - https://authenticationtest.com.*
+    authentication:
+      method: browser
+      parameters:
+        loginPageUrl: https://authenticationtest.com/totpChallenge/
+        loginPageWait: 2
+        browserId: firefox-headless
+        steps:
+        - description: TOTP
+          type: TOTP_FIELD
+          cssSelector: "#totpmfa"
+          value: 123456
+          timeout: 1000
+      verification:
+        method: response
+        loggedInRegex: loginSuccess|mode=logout
+        pollFrequency: 60
+        pollUnits: requests
+        pollUrl: ""
+        pollPostData: ""
+    sessionManagement:
+      method: cookie
+    technology: {}
+    structure: {}
+    users:
+    - name: testuser
+      credentials:
+        totp:
+          secret: I65VU7K5ZQL7WB4E
```

Comment:
It would be preferable to use an env var.

Reply:
Okay that's a good point. Can I tackle that in a future PR because that will mean changing the runner script and figuring out how we want to handle that. Likely we just add it to the mega secret, but still I'd like to discuss/plan.